optaplanner-docs/src/main/docbook/en-US/Chapter-Design_patterns/Chapter-Design_patterns.xml

```diff
@@ -228,34 +226,34 @@
 
           <itemizedlist>
             <listitem>
-              <para>A deterministic gap can be subjected complex business logic. For example in vehicle routing, a
-              cross-continent truck driver needs to rest 15 minutes after 2 hours of driving (which can be during
-              loading/unloading time at a customer location) and also needs to rest 10 hours after 14 hours of
+              <para>A deterministic gap can be subjected to complex business logic. For example in vehicle routing, a
+              cross-continent truck driver needs to rest 15 minutes after 2 hours of driving (which may also occur during
+              loading or unloading time at a customer location) and also needs to rest 10 hours after 14 hours of
               work.</para>
             </listitem>
           </itemizedlist>
         </listitem>
 
         <listitem>
-          <para>Planning variable gaps: This is uncommon, because this extra planning variable (which impacts <link
-          linkend="searchSpaceSize">the search space</link>) reduces efficiency and scalability.</para>
+          <para>Planning variable gaps: This is uncommon, because such extra planning variable (which impacts the <link
+          linkend="searchSpaceSize">search space</link>) reduces efficiency and scalability.</para>
         </listitem>
       </itemizedlist>
     </section>
   </section>
 
   <section xml:id="multiStragePlanning">
-    <title>Multi stage planning</title>
+    <title>Multi-stage planning</title>
 
     <para>For practical or organizational reasons (such as Conway's law), complex planning problems are often broken
     down in multiple stages. A typical example is train scheduling, where one department decides where and when a train
-    will arrive/depart, and another departments assigns the operators and actual train cars/locomotives.</para>
+    will arrive or depart, and another departments assigns the operators to the actual train cars/locomotives.</para>
 
     <para>Each stage has its own solver configuration (and therefore its own <literal>SolverFactory</literal>). Do not
-    confuse with <link linkend="solverPhase">multi phase solving</link> which uses only 1 solver configuration.</para>
+    confuse it with <link linkend="solverPhase">multi-phase solving</link> which uses a one-solver configuration.</para>
 
-    <para>Just like <link linkend="partitionedSearch">Partitioned Search</link>, multi stage planning leads to
-    suboptimal results. Nevertheless, in complex cases it's often worth it to simplify maintenance and ownership, as
-    well as to get the project off the ground.</para>
+    <para>Similarly to <link linkend="partitionedSearch">Partitioned Search</link>, multi-stage planning leads to
+    suboptimal results. Nevertheless, it may be beneficial in order to simplify the maintenance, ownership, and help a project
+    to succeed.</para>
```

Comment:
It's not so much about getting the project to succeed, as it is to get the project started, get it out the gate, get it off the ground, basically avoid "analysis paralysis".